sql/init.sql

```diff
@@ -0,0 +1,89 @@
+-- Create User table
+CREATE TABLE IF NOT EXISTS "User" (
+  "userId" SERIAL PRIMARY KEY,
+  "firstName" VARCHAR(255),
+  "lastName" VARCHAR(255),
+  "email" VARCHAR(255) UNIQUE,
+  "password" VARCHAR(255),
+  "streetAddress" VARCHAR(255),
+  "building" VARCHAR(255),
+  "zipCode" VARCHAR(255),
+  "city" VARCHAR(255),
+  "country" VARCHAR(255),
+  "phone" VARCHAR(255),
+  "createdAt" TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP,
+  "updatedAt" TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP,
+  CONSTRAINT "User_email_unique" UNIQUE ("email")
+);
+
+-- Create Category table
+CREATE TABLE IF NOT EXISTS "Category" (
+  "id" SERIAL PRIMARY KEY,
+  "name" VARCHAR(255),
+  "createdAt" TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP,
+  "updatedAt" TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Create Tax table
+CREATE TABLE IF NOT EXISTS "Tax" (
+  "taxId" SERIAL PRIMARY KEY,
+  "zipCode" VARCHAR(255),
+  "percentage" FLOAT,
+  "createdAt" TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP,
+  "updatedAt" TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Create Product table
+CREATE TABLE IF NOT EXISTS "Product" (
+  "id" SERIAL PRIMARY KEY,
+  "categoryId" INT,
+  "name" VARCHAR(255),
+  "description" VARCHAR(255),
+  "brand" VARCHAR(255),
+  "image" VARCHAR(255),
```

Comment:
I saw it is saving image URLs, is not better to call it as it is? Also, ensure that the URLs you store are valid and accessible. You might want to periodically check the validity of URLs to avoid broken links